proposals/smi.adoc

```diff
@@ -0,0 +1,123 @@
+== Service Mesh Interface Proposal
+*Name of Project:*
+Service Mesh Interface
+
+*Project Description:*
+Service Mesh Interface is an open source specification for interoperable service meshes on Kubernetes. Launched in May 2019, SMI brings the ecosystem together with a standard which enables innovation by providing a common abstraction that anyone can build against.
+
+*Statement on alignment with CNCF charter mission:*
+The Service Mesh Interface spec is well-aligned with the CNCF mission to make cloud native computing ubiquitous. Interoperability creates a virtuous cycle which increases adoption. SMI encourages an open source, vendor-neutral service mesh ecosystem by empowering end users to build upon ecosystem tools without needing to re-implement the underlying substrate.
+
+*Comparison with similar projects (inside or outside the CNCF), including what differentiates this project:*
+SMI is complementary to existing CNCF projects like LinkerD; it encourages adoption of service mesh. SMI is the only cross-industry effort to make service meshes interoperable. For example, LinkerD is one example of a service mesh that implements and uses SMI.
```

Comment:
`SMI is the only cross-industry effort to make service meshes interoperable` isn't accurate. There is also [Hamlet](https://octo.vmware.com/project-hamlet-secure-multi-vendor-multi-mesh-federation-open-source/). From the `Closing Remarks` section of that announcement:

```
Closing Remarks

Through Hamlet and the collaborative effort of the teams involved within Istio, Google Cloud’s Anthos, Hashicorp Consul, Pivotal, and VMware NSX Service Mesh, service meshes can now interoperate, enabling customers to discover and securely communicate services across these meshes.
```

Reply:
Hamlet hasn't had activity in the last ~4 months so maybe it's a dead project now? Prior art should still be mentioned though

Reply:
@sergiopozoh can add more information on the maintenance status of Hamlet. My understanding is Hamlet continues to be a running project.

Cheers
-steve

Reply:
Added callout to Hamlet in **Comparison with similar projects** section